shared/appengine.go

```diff
@@ -185,6 +181,15 @@ func (a appEngineAPIImpl) GetRunsURL(filter TestRunFilter) *url.URL {
 	return getURL(a.GetHostname(), "/runs", filter)
 }
 
+func (a appEngineAPIImpl) GetWPTFYIGithubBot() (*github.Client, error) {
```

Comment:
I note that `GetGitHubClient` caches the client, whereas this version just retrieves it each time. That may be a reasonable choice given usage, just wanted to make sure we were aware of the difference.

Reply:
Right, we can look at the /api/metadata usage on the dashboard after it launches.